content/messaging/index.mdx

```diff
@@ -46,12 +46,25 @@ Here are some general rules, which summarize the previous section and add new in
 
 ### The flow
 
-To better understand the flow between services, watch a short video.
+Watch a short video that illustrates the basic flow between services at LiveChat.
 
-<video width="750" height="500" style="height: auto;" controls>
+<video width="750" height="500" style="height: auto;" style="border-bottom: 20px solid white;" controls>
   <source type="video/mp4" src="images/livechat-messaging-api-flow.mp4" />
 </video>
 
+For more technical details, read the description below.
+
+![Basic flow between services at LiveChat](images/basic-flow-between-services-livechat.png)
+
+The LiveChat communication protocol uses *websockets*. They act like tubes that connect two parts and enable transport between them. In the basic model of the LiveChat services, there are two websocket connections:
+
+- Agent App --- LiveChat's backend
```

Comment:
These characters: `---` were misleading for me, I tried to understand why you call agent app as Livechat's backend :D